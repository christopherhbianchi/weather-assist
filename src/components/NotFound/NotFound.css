.notFoundMessage{
  margin: 0;
  padding-top: 130px;
  text-align: center;
  color: #87b1f7;
}
