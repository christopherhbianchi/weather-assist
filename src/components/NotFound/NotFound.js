import React from 'react';
import './NotFound.css';

export const NotFound = props => {
  return(
    <div>
      <h2 className='notFoundMessage'>Resource not found.</h2>
    </div>
  );
};

export default NotFound;
