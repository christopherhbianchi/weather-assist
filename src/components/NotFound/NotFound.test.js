import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound.js';

describe('<NotFound />', () => {
  it('should match snapshot', () => {
    const renderedComponent = shallow(<NotFound />);
    expect(renderedComponent).toMatchSnapshot();
  });//it
});//describe
