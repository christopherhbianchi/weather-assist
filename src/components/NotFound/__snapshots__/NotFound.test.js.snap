// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h2
        className="notFoundMessage"
>
        Resource not found.
</h2>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Resource not found.",
        "className": "notFoundMessage",
      },
      "ref": null,
      "rendered": "Resource not found.",
      "type": "h2",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h2
          className="notFoundMessage"
>
          Resource not found.
</h2>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Resource not found.",
          "className": "notFoundMessage",
        },
        "ref": null,
        "rendered": "Resource not found.",
        "type": "h2",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
